src/sentry/integrations/gitlab/client.py

```diff
@@ -95,9 +99,17 @@ def request(self, method, path, data=None, params=None):
         )
 
     def get_user(self):
+        """Get a user
+
+        See https://docs.gitlab.com/ee/api/users.html#single-user
+        """
```

Comment:
I don't know if we do this all over, but I appreciate you linking to the docs for each request.